<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>William Findlay</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ebpH</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/assets/img/logos/favicon.png</normaloff>:/img/assets/img/logos/favicon.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QPlainTextEdit" name="event_log">
      <property name="statusTip">
       <string>ebpH event log</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="backgroundVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ebpH Log</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>933</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>&amp;Export</string>
     </property>
     <addaction name="actionExport_Logs"/>
     <addaction name="actionExport_Statistics"/>
    </widget>
    <addaction name="action_Force_Save_Profiles"/>
    <addaction name="menuExport"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="action_Preferences"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionebpH_Help"/>
    <addaction name="separator"/>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menu_Actions">
    <property name="title">
     <string>&amp;Monitoring</string>
    </property>
    <addaction name="action_Start_Monitoring"/>
    <addaction name="action_Stop_Monitoring"/>
    <addaction name="separator"/>
    <addaction name="action_View_Modify_Profile"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Actions"/>
   <addaction name="menu_Settings"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Start_Monitoring"/>
   <addaction name="action_Stop_Monitoring"/>
   <addaction name="separator"/>
   <addaction name="action_View_Modify_Profile"/>
   <addaction name="separator"/>
   <addaction name="action_Quit"/>
  </widget>
  <action name="action_About">
   <property name="text">
    <string>&amp;About ebpH</string>
   </property>
  </action>
  <action name="actionebpH_Help">
   <property name="text">
    <string>ebpH &amp;Help</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/assets/img/icons/delete.png</normaloff>:/img/assets/img/icons/delete.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="statusTip">
    <string>Exit the application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Start_Monitoring">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/assets/img/icons/eye.png</normaloff>:/img/assets/img/icons/eye.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Start Monitoring</string>
   </property>
   <property name="statusTip">
    <string>Start monitoring processes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_Stop_Monitoring">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/assets/img/icons/hide.png</normaloff>:/img/assets/img/icons/hide.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Stop Monitoring</string>
   </property>
   <property name="statusTip">
    <string>Stop monitoring processes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_View_Modify_Profile">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/assets/img/icons/browser.png</normaloff>:/img/assets/img/icons/browser.png</iconset>
   </property>
   <property name="text">
    <string>View/Modify &amp;Profile</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_Force_Save_Profiles">
   <property name="text">
    <string>Force &amp;Save Profiles</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Preferences">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
  <action name="actionExport_Logs">
   <property name="text">
    <string>Export &amp;Logs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E, Ctrl+L</string>
   </property>
  </action>
  <action name="actionExport_Statistics">
   <property name="text">
    <string>Export &amp;Statistics</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E, Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
